--- libmpdclient-2.3/src/song.c.org	2010-10-03 04:17:32.563828505 +0200
+++ libmpdclient-2.3/src/song.c	2010-10-03 04:18:14.767763969 +0200
@@ -116,7 +116,8 @@ mpd_song_new(const char *uri)
 		return NULL;
 	}
 
-	for (unsigned i = 0; i < MPD_TAG_COUNT; ++i)
+    unsigned i;
+	for (i = 0; i < MPD_TAG_COUNT; ++i)
 		song->tags[i].value = NULL;
 
 	song->duration = 0;
@@ -137,8 +138,9 @@ void mpd_song_free(struct mpd_song *song
 	assert(song != NULL);
 
 	free(song->uri);
-
-	for (unsigned i = 0; i < MPD_TAG_COUNT; ++i) {
+    
+    unsigned i;
+	for (i = 0; i < MPD_TAG_COUNT; ++i) {
 		struct mpd_tag_value *tag = &song->tags[i], *next;
 
 		if (tag->value == NULL)
@@ -177,8 +179,9 @@ mpd_song_dup(const struct mpd_song *song
 	if (ret == NULL)
 		/* out of memory */
 		return NULL;
-
-	for (unsigned i = 0; i < MPD_TAG_COUNT; ++i) {
+    
+    unsigned i;
+	for (i = 0; i < MPD_TAG_COUNT; ++i) {
 		const struct mpd_tag_value *src_tag = &song->tags[i];
 
 		if (src_tag->value == NULL)
